src/components/Slider/Slider.tsx

```diff
@@ -0,0 +1,241 @@
+import React, { FC, useLayoutEffect, useRef, useState } from 'react';
+
+import { mergeClasses } from '../../shared/utilities';
+import styles from './slider.module.scss';
```

Comment:
For consistency, please move sass module import after all other imports.

Reply:
Roger... also the result of Typescript Hero sorting. I'll investigate its configurability.